shared/js/ui/templates/shared/hero.es6.js

```diff
@@ -0,0 +1,18 @@
+const bel = require('bel')
+
+module.exports = function (ops) {
+  return bel`<div class="hero border--bottom text--center">
+    <a href="#" class="hero__close js-sliding-subview-close">
+      <span class="icon icon__arrow icon__arrow--large icon__arrow--left">
+      </span>
+    </a>
+    <div class="hero__icon hero__icon--${ops.status}">
+    </div>
+    <h1 class="hero__title">
+      ${ops.title}
+    </h1>
+    <h2 class="hero__subtitle">
+      ${ops.subtitle}
+    </h2>
+  </div>`
+}
```

Comment:
cc @MariagraziaAlastra here's a hero template you can now use. You'll likely need to override some styles to set the individual styles, e.g. https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/94e66350f6792b05e892d509ed098c47c77ee353/shared/scss/views/_privacy-practices.scss#L14-L28